COLLADA2GLTF/GLTF/GLTFAsset.cpp

```diff
@@ -941,29 +941,54 @@ namespace GLTF
         }
         
         meshesUIDs = meshes->getAllKeys();
+		shared_ptr <GLTF::JSONObject> materials = this->_root->createObjectIfNeeded(kMaterials);
```

Comment:
It looks like your editor is inserting tabs instead of spaces. In Visual Studio go to Tools->Options->Text Editor->All Languages->Tabs. Select `Insert Spaces` and set `Tab Size